### PR TITLE
Changes according the new version of codechef

### PR DESCRIPTION
**Pull request by pvskand**
I guess this was compatible with the earlier version of codechef, but as of now its not. So I had done some changes accordingly to make it compatible. Issue #9 was also due to the same reason.

### CODE REVIEW

File: fastsubmit.py [hunk 1]
```diff
@@ -48,18 +55,26 @@ def f_submit(site,langlist):
...
-        parser = BeautifulSoup(str(response.read()))
-        div_list = parser.find_all('div', class_='head')
-        for divs in div_list:
-                for content in divs.contents:
-                        if str(content).find('Status') is not -1:
-                                status = str(content).split(',')[0]
-                                return status.split(':')[1]
+
+        #response = mechanize.urlopen('http://www.codechef.com/viewsolution/'+solution_id)
+        site= 'http://www.codechef.com/viewsolution/'+solution_id
```
Comment (dhruvagarwal) [inline fastsubmit.py]: The indent seems to be inconsistent

File: fastsubmit.py [hunk 2]
```diff
@@ -71,8 +86,11 @@ def cc():
                             solution_id = tokens[3].split(';')[0]
                             print "Running your solution. Sit tight."
                             res=recheck_cc(solution_id)
-                            while '??' in res:
+                            while True:
+                            	if res == -1:
                                     res=recheck_cc(solution_id)
+                                else:
+                                	break
```
Comment (dhruvagarwal) [inline fastsubmit.py]: Use 4 spaces

File: fastsubmit.py [hunk 3]
```diff
@@ -33,13 +34,19 @@ def ran_string(res, list):
     		d=int(res.strip())
     	except:
     		print random.choice(myList[2])                   
+def select_form(form):
+	if form.attrs.get('id', None) == 'problem-submission':
```
Comment (dhruvagarwal) [inline fastsubmit.py]: I think this is a debug statement?

**Review by dhruvagarwal — CHANGES_REQUESTED** [review]

**Comment by dhruvagarwal** [comment]
Thanks a lot for taking time out. I've given some comments above. Please revert

**Review by dhruvagarwal — APPROVED** [review]

**Comment by dhruvagarwal** [comment]
I hope you've tested this. Can you squash the commits and then update?

**Comment by pvskand** [comment]
I am facing problems in squashing the commits. Its my first time. Are the above changes in squashing the commits fine?

**Comment by dhruvagarwal** [comment]
Its alright @pvskand . I can guide you a bit on this. I'm considering my repo's remote to be `origin`
```
git fetch origin
git rebase -i origin develop
```
Now, you'll see a prompt just replace all the `pick` with `f` but the first one.
Save the prompt. You'll see a success message. 
Now, you'll have to force push to your branch.

The idea of squashing is to reduce the commits so that the git history is short, to the point and sweet :)

**Comment by pvskand** [comment]
Thanks @dhruvagarwal. I guess its fine now. btw I have tested the code.

**Comment by dhruvagarwal** [comment]
Thanks for your contribution :)

**Pull request merged**